modules/internal/ChapelDistribution.chpl

```diff
@@ -581,7 +587,19 @@ module ChapelDistribution {
     // atomics are available
     var _arrAlias: BaseArr;    // reference to base array if an alias
     var pid:int = nullPid; // privatized ID, if privatization is supported
+
+    proc isSliceArrayView() param {
+      return false;
+    }
+
+    proc isRankChangeArrayView() param {
+      return false;
+    }
   
+    proc isReindexArrayView() param {
+      return false;
+    }
```

Comment:
Regarding point two: It was my mistake to use canResolve, and I should have been using these methods to begin with.